src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -473,6 +475,8 @@ private static void EnsureNullableAttributeExists(CSharpCompilation compilation,
                         break;
 
                     case SyntaxKind.OutKeyword:
+                        Debug.Assert(!seenReadonly);
```

Comment:
>Debug.Assert(!seenReadonly); [](http://example.com/codeflow?start=24&length=28)

I am not sure why this should be the case. We can assert that ```seenReadonly``` is false unless ```seenRef``` is true, but I do not think we can assume that  ```seenReadonly``` is always false here. We probably should add a test that demonstrates that.